### PR TITLE
Adding 1.1.0 documentation to the website

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

File: docs/fluo/1.1.0-incubating/applications.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+## Creating a Fluo observer
+
+To create an observer, follow these steps:
+
+1. Create one or more classes that extend [Observer] like the example below. Please use [slf4j] for
+   any logging in observers as [slf4j] supports multiple logging implementations. This is
+   necessary as Fluo applications have a hard requirement on [logback] when running in YARN.
+
+    ```java
```
Comment (keith-turner) [inline docs/fluo/1.1.0-incubating/applications.md]: I don't know why, but this did not render properly when I ran Jekyll locally.

**Review by keith-turner — APPROVED** [review]

File: docs/fluo/1.1.0-incubating/applications.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    class AppObserverProvider implements ObserverProvider {
+      @Override
+      public void provide(Registry or, Context ctx) {
+        //setup InvertObserver to be triggered when the column obs:data is modified
+        or.forColumn(new Column("obs", "data"), NotificationType.STRONG)
+          .useObserver(new InvertObserver());
+        
+        //Observer is a Functional interface.  So Obsevers can be written as lambdas.
```
Comment (cjmctague) [inline docs/fluo/1.1.0-incubating/applications.md:118]: Obsevers should be Observers

File: docs/fluo/1.1.0-incubating/applications.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+        or.forColumn(new Column("new","data"), NotificationType.WEAK)
+          .useObserver((tx,row,col) -> {
+             Bytes combined = combineNewAndOld(tx,row);
+             tx.set(row, new Column("current","data"), combined);
+           });
+      }
+    }
+    ```
+
+3.  Build a jar containing thses classes and include this jar in the `lib/` directory of your Fluo
```
Comment (cjmctague) [inline docs/fluo/1.1.0-incubating/applications.md:128]: thses should be theses

File: docs/fluo/1.1.0-incubating/applications.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+    application.
+4.  Configure your Fluo instance to use this observer provider by modifying the Observer section of
+    [fluo.properties].
+5.  Initialize Fluo.  During initialization Fluo will obtain the observed columns from the 
+    ObserverProvider and persist the columns in Zookeeper.  These columns persisted in Zookeeper
```
Comment (cjmctague) [inline docs/fluo/1.1.0-incubating/applications.md:133]: Is persist the right word here? Do you mean inserted or added to Zookeeper or continued in Zookeeper?

File: docs/fluo/1.1.0-incubating/applications.md (hunk 2 above)
Comment (mikewalch) [reply docs/fluo/1.1.0-incubating/applications.md:118]: Merged before this comment.  Fixed in db72ef51ffb1110f1

File: docs/fluo/1.1.0-incubating/applications.md (hunk 3 above)
Comment (mikewalch) [reply docs/fluo/1.1.0-incubating/applications.md:128]: Fixed in db72ef51ffb1110f1

File: docs/fluo/1.1.0-incubating/applications.md (hunk 4 above)
Comment (mikewalch) [reply docs/fluo/1.1.0-incubating/applications.md:133]: I think the current text is OK. I think @keith-turner wrote this originally so maybe he can weigh in. This PR is to copy the documentation from the main Fluo repo to website for the release.

**Pull request merged**